Здравствуйте!
pydotplus установил, при вводе команды вылетает ошибка:
pydotplus.graphviz.InvocationException: GraphViz's executables not found
Установил отдельно graphviz - та же ошибка

Установил debug-toolbar.
Все сделал как вы. Команда : python manage.py runserver 0.0.0.0:8000
Сервер запустился но не открывается: "Не получается получить доступ к сайту".
Пробовал запускать другими способами - панель не появляется в браузере.
Все обучение приходиться бороться с виндой. Понимаю теперь почему нужен линукс.

Пробовал установить locust. Не может установить пакет psutil. Я боролся с подобной проблемой
скачиванием пакетов под windows. С Вашего позволения остановлюсь на этом.
Два дня убил на этот урок(((